docs/contributing/Building, Debugging, and Testing on Unix.md

```diff
@@ -8,8 +8,18 @@ Particularly for developers who aren't experienced with .NET Core development on
 1. Run `./build.sh --restore`
 1. Run `./build.sh --build`
 
+## Working in Visual Studio Code
+1. Install [VS Code](https://code.visualstudio.com/Download)
+    - After you install VS Code, install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+    - Important tip: You can look up editor commands by name by hitting *Ctrl+Shift+P*, or by hitting *Ctrl+P* and typing a `>` character. This will help you get familiar with editor commands mentioned below. On a Mac, use *⌘* instead of *Ctrl*.
+2. Install a recent preview [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core). At time of writing, Roslyn uses .NET 5 preview 7. The exact version in use is recorded in our [global.json](https://github.com/dotnet/roslyn/blob/master/global.json) file.
+3. You can build from VS Code by running the *Run Build Task* command, then selecting an appropriate task such as *build* or *build current project* (the latter builds the containing project for the current file you're viewing in the editor).
+3. You can run tests from VS Code by opening a test class in the editor, then using the *Run Tests in Context* (**Ctrl+R, T** by default) and *Debug Tests in Context* (**Ctrl+R, Ctrl+T** by default) editor commands.
```

Comment:
These shortcuts were removed for conflicting with the default `CTRL+R` mapping in vscode's core shortcuts. They're currently unmapped.

Reply:
```suggestion
3. You can run tests from VS Code by opening a test class in the editor, then using the *Run Tests in Context* and *Debug Tests in Context* editor commands. You may want to bind these commands to keyboard shortcuts that match their Visual Studio equivalents (**Ctrl+R, T** for *Run Tests in Context* and **Ctrl+R, Ctrl+T** for *Debug Tests in Context*).
```

Bummer! I do actually recall having to re-bind these after updating VS Code.